packages/edit-site/src/components/global-styles/palette.js

```diff
@@ -87,8 +87,7 @@ function Palette( { name } ) {
 			{ window.__experimentalEnableColorRandomizer &&
 				themeColors?.length > 0 && (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> color --> pallete, option for randmoize colors.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="radomize-colors-before" src="https://github.com/user-attachments/assets/d0ae84de-ca07-48c9-92c4-1f3b0c0eea5b"> | <img width="625" alt="randomize-colors-after" src="https://github.com/user-attachments/assets/bb065228-8e19-4e8e-bdc6-b7302f4fa440"> |